docker-hub/image-access-management.md

```diff
@@ -39,19 +39,50 @@ To ensure that each org member uses images in a safe and secure environment, you
-  
-2. On Windows, create a file `C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (i.e., only the system administrator can write to the file). The file must be JSON and contain one or more organization names in the `allowedOrgs` key. The user must sign in and be a member of at least one of the organizations before using Docker Desktop. For example:
 
-    ```json
+2. Create a registry.json file on Windows or Mac.
+
+For **Windows**:
+
+Create a file `C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (i.e., only the system administrator can write to the file). The file must be JSON and contain one or more organization names in the `allowedOrgs` key.
+
+To create your registry.json file:
+
+- Open Windows Powershell and select Run as Administrator.
+- Type the following command: `cd /ProgramData/DockerDesktop/`
+- In notepad, type `registry.json` and enter one or more organization names in the `allowedOrgs` key and click Save.
+
+Example:
+
+        ```json
+     {
+       "allowedOrgs": ["mycompany"]
+     }
+     ```
+- Navigate to Powershell and type ```start .```
+
+> You have just created the registry.json file.
+
+For **macOS**:
+
+Create a file `/Library/Application Support/com.docker.docker/registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (i.e., only the system administrator can write to the file). The file must be JSON and contain one or more organization names in the `allowedOrgs` key. The user must sign in and be a member of at least one of the organizations before using Docker Desktop.
+
+To create your registry.json file:
+- Navigate to VS Code or any text editor of your choice.
+- Enter one or more organization names in the `allowedOrgs` key and save it in your Documents.
+
+Example:
+
+       ```json
     {
       "allowedOrgs": ["mycompany"]
     }
     ```
 
-3. On macOS, create a file `/Library/Application Support/com.docker.docker/registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (i.e., only the system administrator can write to the file). The file must be JSON and contain one or more organization names in the `allowedOrgs` key. The user must sign in and be a member of at least one of the organizations before using Docker Desktop. For example: 
+- Open a new terminal and type the following command: `sudo mkdir -p /Library/Application\ Support/com.docker.docker`
 
-    ```json
-    {
-      "allowedOrgs": ["mycompanymarketing", “mycompanydev”]
-    }
-    ```
+> If prompted, type your password associated with your local computer.
```

Comment:
Not sure if the intention was to format these as a Note as these are also part of the instructions?  They currently appear as

![image](https://user-images.githubusercontent.com/47779042/135661957-fbf21877-ebff-44d8-9808-9bd0b0dfdfcd.png).